src/common/java/net/minecraftforge/gradle/common/util/runs/RunConfigGenerator.java

```
@@ -356,11 +352,14 @@ public final void createRunConfiguration(@Nonnull final MinecraftExtension minec
                 transformer.setOutputProperty("{http://xml.apache.org/xslt}indent-amount", "2");
 
                 minecraft.getRuns().forEach(runConfig -> {
-                    final Map<String, Document> documents = createRunConfiguration(project, runConfig, docBuilder, additionalClientArgs);
+                    final Map<String, Document> documents = createRunConfiguration(minecraft, project, runConfig, docBuilder, additionalClientArgs);
 
                     documents.forEach((fileName, document) -> {
                         final DOMSource source = new DOMSource(document);
-                        final StreamResult result = new StreamResult(new File(runConfigurationsDir, fileName));
+                        final File location = new File(runConfigurationsDir, fileName);
+                        if (!location.getParentFile().exists())
+                            location.getParentFile().mkdirs();
```

Should we throw an exception if `mkdirs()` returns false?